expectise/expect.py

```diff
@@ -215,7 +220,7 @@ def and_raise(self, error: Exception) -> Expect:
         return self
 
     @classmethod
-    def set_up(cls, klass: Type, method_name: str, surrogate: Callable) -> None:
+    def set_up(cls, klass: Type, method_name: str, surrogate: Callable, mock_ref: Mock) -> None:
```

Comment:
I would suggest to change the function prototype to 
```python
def set_up(cls, mock: Mock) -> None:
    ...
```

since you can retrieve all other attributes from the `Mock` instance

Reply:
Good point, done!